test/passing/let_binding.ml

```diff
@@ -1,34 +1,34 @@
 (* Note that {[ let ident : typ = exp ]} is different from {[ let (ident :
    typ) = exp ]}. The difference should be maintained *)
 
-let (_ : int) = x1
+let _ : int = x1
 
 let (x : int) = x2
 
-let (_ : int) = x3
+let _ : int = x3
 
 let x : int = x4
 
 let _ =
   let (x : int) = x in
   let x : int = x in
-  let (_ : int) = x in
-  let (_ : int) = x in
+  let _ : int = x in
```

Comment:
I think tests like this should be added to a file where the added option is set.

Also, it is cases like this that make me think that this option is confusing, as it seems to format in a way that obscures the underlying parsetree. It does not actually, but only because `_` is not technically an identifier, although just about every programmer thinks of it as an identifier.

Reply:
There should also be tests involving extension points:
```
  let (x : int) = x in
  let x : int = x in
  let (_ : int) = x in
  let _ : int = x in
  let%ext (x : int) = x in
  let%ext x : int = x in
  let%ext (_ : int) = x in
  let%ext _ : int = x in
```